api/walletApi-V0.yml

```diff
@@ -191,17 +201,14 @@ paths:
           description: Unprocessable Entity - syntax is correct, but the server was unable to process the contained instructions.
           schema:
             $ref: "#/definitions/Error"
-        "423":
-          description: Locked - The resource that is being accessed is locked
-          schema:
-            $ref: "#/definitions/Error"
         "500":
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
             $ref: "#/definitions/Error"
-  /rest/wallet/{nickname}/signOperation:
+  /rest/accounts/{nickname}/signOperation:
```

Comment:
because:
- I don't understand this locked concept, I poinged Soleneand maybe she will help me
- because is that situation we ask the end user to authenticate himself and the authntication failed. being unauthorized is natural.

Reply:
okay. After reading more information about 423:

"
A major goal of WebDAV was to provide a ‘filesystem over the web’. One of its core features is for users to ‘lock’ specific files and directories to prevent others from editing them.

A user can lock a resource by issuing a http [LOCK](https://tools.ietf.org/html/rfc4918#section-9.10) method and later on unlock the resource by using the [UNLOCK](https://tools.ietf.org/html/rfc4918#section-9.11) HTTP method.
", https://evertpot.com/http/423-locked

423 is not appropriate. I'm changing to 401